index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Singolo</title>
+    <link rel="stylesheet" href="my_css.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <header>
+        <div class="header_wrapp">
+            <div class="wrapp_logo_container">
+                <h1 class="h1_wrapp">singolo<span class="h1_wrapp_logo">*</span></h1>
+            </div>
+            <nav class="header_nav">
+                <ul class="header_nav_row">
+                    <li><a href="#">home</a></li>
+                    <li><a href="#">services</a></li>
+                    <li><a href="#">portfolio</a></li>
+                    <li><a href="#">about</a></li>
+                    <li><a href="#">contact</a></li>
+                </ul>
+            </nav>
+        </div>
+    </header>
+    <div class="slider">
+        <div class="slider_wrapp">
+            <div class="slider_controls">
+                <div class="slider_controls_arrow_left">
+                    <figure class="img_slider_controls_arrow_left">
+                        <img src="./img/slider-controls/arrow-left.png" alt="slider_controls_arrow_left">
+                    </figure>
+                </div>
+                <div class="slider_controls_arrow_right">
+                    <figure class="img_slider_controls_arrow_right">
+                        <img src="./img/slider-controls/arrow-right.png" alt="slider_controls_arrow_right">
+                    </figure>
+                </div>
+            </div>
+            <div class="slider_content">
+                <div class="slider_content_phone_vertical">
+                    <figure>
+                        <img src="./img/phone-vertical.png" alt="slider_content_phone_vertical">
+                    </figure>
+                </div>
+                <div class="slider_content_phone_horizontal">
+                    <figure>
+                        <img src="./img/phone-horizontal.png" alt="slider_content_phone_horizontal">
+                    </figure>
+                </div>
+            </div>
+        </div>
+    </div>
+</body>
+
+</html>
```

Comment:
Please add new line at the end of file
See more https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file